src/components/CreateAnalysis/CreateAnalysisDetail.tsx

```diff
@@ -68,17 +89,20 @@ const CreateAnalysisDetail: React.FC<CreateAnalysisDetailProps> = (props) => {
               <div className="numberCircle">{numOfSelectedImages}</div>
               <h3>Series selected</h3>
               <a onClick={() => props.setIsExpanded(true)}>(More details)</a>
-              <RightArrowButton click={submitAnalysis}>
+              <RightArrowButton click={submitAnalysis} XrayModel={XrayModel} CTModel={CTModel}>
```

Comment:
Instead of passing `XrayModel` and `CTModel` down to `RightArrowButton`, I suggest creating a wrapper function that invoked `submitAnalysis` with the right parameters.

One way to do it is
```suggestion
              <RightArrowButton click={() => {submitAnalysis(XrayModel, CTModel)}}>
```

Please, look at the comment in `RightArrowButton.tsx` to see why this may be a cleaner approach.